cfme/fixtures/provider.py

```diff
@@ -382,7 +382,7 @@ def big_template_modscope(provider):
     return _get_template(provider, 'big_template')
 
 
-@pytest.fixture(scope="function")
+@pytest.fixture(scope="module")
```

Comment:
@digitronik, @mshriver, we need to be more accurate with changing scope of global fixtures. this PR broke 50 - 100 tests due to fixture scope mismatch. I'm reverting these changes - https://github.com/ManageIQ/integration_tests/pull/8736

and let me add that if we need to change something global, we must add all tests affected by this change.

@digitronik, please make sure your test isn't ok after scope fix

Reply:
@izapolsk i changed on basis of theory that `higher` scope fixture can used within `lower` scope. 

Ahhh understand; might be test which are using this has `provider` in `function` scope.
my bad; i'm not checked all service or other tests which are using this fixtures. ok; I will seat with @niyazRedhat for his test structure. `Provider` `function` scope increasing automation time... yup, i know still it will behave like function scope.
Thanks for quick resolution.